.github/workflows/main.yml

```diff
@@ -7,8 +10,17 @@ on:
     branches: [develop]
 
 jobs:
-  build:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@20.8b1
+        with:
+          args: ". --check"
```

Comment:
This is not required because it will also check without this.

Reply:
@codesankalp I know it's not necessary but it's mentioned in the official docs so added it , do confirm if it's to be removed